test/libsolidity/smtCheckerTests/operators/conditional_assignment_3.sol

```diff
@@ -0,0 +1,12 @@
+pragma experimental SMTChecker;
+
+contract C {
+	function f(uint a, uint b) public pure {
+		require(a > 10);
+		require(b < 3);
+		uint c = (a > 10) ? b : a;
```

Comment:
Here it needs to report always true as there is `require(a > 10)` in the beginning of the function

Reply:
Solved by adding `ConstantCondition` verification target in BMC